src/code42cli/cmds/users.py

```diff
@@ -84,6 +93,95 @@ def remove_role(state, username, role_name):
     _remove_user_role(state.sdk, role_name, username)
 
 
+@users.command(name="update")
+@user_uid_option
+@click.option("--username", help="The new username for the user.")
+@click.option("--password", help="The new password for the user.")
+@click.option("--email", help="The new email for the user.")
+@click.option("--first-name", help="The new first name for the user.")
+@click.option("--last-name", help="The new last name for the user.")
+@click.option("--notes", help="Notes about this user.")
+@click.option(
+    "--archive-size-quota", help="The total size (in bytes) allowed for this user."
+)
+@sdk_options()
+def update_user(
+    state,
+    user_id,
+    username,
+    email,
+    password,
+    first_name,
+    last_name,
+    notes,
+    archive_size_quota,
+):
+    """Update a user with the specified unique identifier."""
+    _update_user(
+        state.sdk,
+        user_id,
+        username,
+        email,
+        password,
+        first_name,
+        last_name,
+        notes,
+        archive_size_quota,
+    )
+
+
+_bulk_user_update_headers = [
+    "user_id",
+    "username",
+    "email",
+    "password",
+    "first_name",
+    "last_name",
+    "notes",
+    "archive_size_quota",
+]
+
+
+@users.group(cls=OrderedGroup)
+@sdk_options(hidden=True)
+def bulk(state):
+    """Tools for managing users in bulk"""
+    pass
+
+
+users_generate_template = generate_template_cmd_factory(
+    group_name="users",
+    commands_dict={"update": _bulk_user_update_headers},
+    help_message="Generate the CSV template needed for bulk user commands.",
+)
+bulk.add_command(users_generate_template)
+
+
+@bulk.command(name="update")
+@read_csv_arg(headers=_bulk_user_update_headers)
+@format_option
+@sdk_options()
+def bulk_update(state, csv_rows, format):
+    """Update a list of users from the provided CSV."""
+    csv_rows[0]["updated"] = False
```

Comment:
`updated` is a boolean here but a string on line 174

Reply:
Updated to use a string here.